### PR TITLE
NRPT-152: add csv import for coors administrative sanctions 

### PR DESCRIPTION
**Pull request by mtCarto**
[NRPT-152](https://bcmines.atlassian.net/browse/NRPT-152)

### CODE REVIEW

File: api/src/importers/coors/admin-sanction-utils.js [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    let legislation;
+    if (!csvRow['regulation_description']) {
+      legislation = csvRow['act']
+    } else {
+      legislation = csvRow['regulation_description']
+    }
+
+    let enforcement = '';
+    if (csvRow['enforcement_licence_code'] === 'DDC') {
+      enforcement = "Director's decision to suspend licence";
```
Comment (BcGovNeal) [inline api/src/importers/coors/admin-sanction-utils.js:133]: `enforcement` value is slightly different than the ticket.  Same for the `else if` block

File: api/src/importers/coors/admin-sanction-utils.js [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+      enforcement = 'Licence action'
+    }
+
+    const section = csvRow['section'] || '';
+    const sub_section = (csvRow['sub_section'] && ` & Sub-Section ${csvRow['sub_section']}`)|| '';
+    const paragraph = (csvRow['paragraph'] && ` & Paragraph ${csvRow['paragraph']}`)|| '';
+    const description = csvRow['violations_prompting_action'] || '';
+    let summary = '';
+    if (sectionNum === 85) {
+      summary = `Licence action resulting from an unpaid fine for an offense under the ${legislation} & Section ${section}${sub_section}${paragraph} - ${description}`;
```
Comment (BcGovNeal) [inline api/src/importers/coors/admin-sanction-utils.js:146]: I don't think the `&` characters are supposed to be part of the final output.  For example, the output currently reads like "Angling and Scientific Collection Regulation & Section 17 & Sub-Section d".  It should probably read like "Angling and Scientific Collection Regulation Section 17 Sub-Section d"

**Review by BcGovNeal — APPROVED** [review]
Other than the 2 comments I had, it works great!

File: api/src/importers/coors/admin-sanction-utils.js (hunk 1 above)
Comment (mtCarto) [reply api/src/importers/coors/admin-sanction-utils.js]: updated to match latest enforcement value

File: api/src/importers/coors/admin-sanction-utils.js (hunk 2 above)
Comment (mtCarto) [reply api/src/importers/coors/admin-sanction-utils.js]: Updated to this format: Wildlife Act 26 (1) (C)

**Review by marklise — APPROVED** [review]

**Pull request merged**